roles/nginxplus/files/conf/http/cdh_prod_prosody.conf

```diff
@@ -0,0 +1,88 @@
+# Ansible managed
+proxy_cache_path /data/nginx/prosody/NGINX_cache/ keys_zone=prosodycache:10m;
+
+upstream prosody {
+    zone prosody 64k;
+    server cdh-prosody1.princeton.edu;
+    sticky learn
+          create=$upstream_cookie_prosodycookie
+          lookup=$cookie_prosodycookie
+          zone=prosodyclient_sessions:1m;
+}
+
+server {
+    listen 80;
+    server_name prosody.princeton.edu;
+
+    location / {
+        return 301 https://$server_name$request_uri;
+    }
+}
+
+server {
+    listen 80;
+    server_name prosody-beta.princeton.edu;
+
+    location / {
+        return 301 https://$server_name$request_uri;
+    }
+}
```

Comment:
do we need to listen to `ppa.princeton.edu` as well? would redirect to `prosody` as the canonical (but don't need this part yet)